### PR TITLE
Check if item.data is a table

### PR DESCRIPTION
**Pull request by przepompownia**
to prevent an error message if it's a number

### CODE REVIEW

**Comment by przepompownia** [comment]
I didn't investigate why `item.data` can be a number. It simply checks the type before use `item.data.funcParensDisabled`.

File: lua/nvim-autopairs/completion/handlers.lua [hunk 1]
```diff
@@ -12,7 +12,7 @@ M["*"] = function(char, item, bufnr, commit_character)
     local char_before, char_after = utils.text_cusor_line(line, col, 1, 1, false)
 
     if char == '' or char_before == char or char_after == char
-        or (item.data and item.data.funcParensDisabled)
+    or (item.data and type(data) == 'table' and item.data.funcParensDisabled)
```
Comment (adalessa) [inline lua/nvim-autopairs/completion/handlers.lua:15]: why data and not `item.data` ?

Comment (przepompownia) [reply lua/nvim-autopairs/completion/handlers.lua]: With this obvious mistake the next part has never been evaluated :grin: Thanks for keeping your eye on!

Comment (adalessa) [reply lua/nvim-autopairs/completion/handlers.lua]: Thanks for finding the place to make the change, I started to use phpactor and got the error and this help me to at least fix it locally.

Comment (przepompownia) [reply lua/nvim-autopairs/completion/handlers.lua]: https://github.com/windwp/nvim-autopairs/issues/274#issuecomment-1264310074 probably it's not a phpactor bug.

**Comment by windwp** [comment]
Thanks

**Pull request merged**